### DISCUSSION TITLE
uhf, comparison with ijv

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#422] [General]
In uhf I have never carried out tests and in any case I very rarely listened, I did a test today;
It seems to have reception difficulties for ISS (437.800), approximately with egzumer you can listen 60% of the time compared to ijv, in the remaining 40% only an indecipherable hiss, It would be interesting if someone could do the test to confirm or deny what is happening to me

**Comment by pietss** [comment]
Update. Further proof, I confirm that with ijv the reception is better because it continues when the ISS is about to exit the window that concerns my position, egzumer receives only when the signal is stronger. There are 2 radios used, for greater safety I will change the firmware on the 2 radios by reversing them, perhaps already this evening but I'm not sure.
I'm not dying to do this but I'm also interested in understanding if a radio is deafer.

**Comment by pietss** [comment]
Test carried out, it seems that a radio is less sensitive on this band (for the other bands I don't know for the moment), tomorrow I will try again, the position of the ISS in this passage was not very favourable, I would like to try with a transit similar to the one 2 hours ago to have a more reliable parameter

**Comment by pietss** [comment]
Last test. Yesterday after the last test I noticed that one battery was a little more discharged, the voltage difference was about 0.5 volts, from the little I know it seems that this can sometimes reduce the sensitivity, so today I equalized the two batteries and proceeded to listen to the iss. The radio with ijv is more sensitive - and as I said yesterday I reversed the two firmwares to exclude that the problem is due to inherent defects - because towards the end of the transit time, and that is presumably when the signal becomes weaker, with egzumer is no longer received. If I'm not boring, maybe I'll proceed with other identical tests, it would be a nice thing if someone who owns two radios could verify and confirm or deny. In the aeronautical band, for an uninterrupted weather transmission, I positioned myself specifically to minimize the signal, I listened alternately with the two radios to use the identical support point, no difference.

**Comment by pietss** [comment]
Last check, I'll end here. Carried out a little while ago, I confirm what I noted before, naturally the squelch is deactivated.
I finished the soliloquy, bye

**Comment by egzumer** [comment]
IJV refuses to share his code, so I'm unable to even take a look for his improvements. Most probably he simply uses different bandwidth settings. I'm not a HAM, I mostly use the radio for handheld to handheld communication and it works good enough for me. I didn't have time or opportunities to test it any deeper.
You can try out https://github.com/kamilsss655/uv-k5-firmware-custom/, this is a fork that offers more bandwidth options. Kamil stated that this offers better reception. You can also try spectrum mode, where you can switch to 6.25kHz BW.

**Reply by BruMcD** [reply]
Bandwidth is most likely the key.  ISS Is an interesting choice of test signal as doppler effects will mean that for me (N Europe) their 437.800000 will first appear to be 437.809 then swing through the right frequency and end up around 437.791 IIRC.  

This range will test even a 25KHz channel spacing 'filter' which is typically only +/-7.5Khz at the shoulders and so result in distortion and/or squelch closure.    If IJV can still receive 437.791 well on 437.800 tuning at the end of the window (but Egzumer cannot) IJV must be using (by setting or otherwise) a wider bandwidth receive filter shape

I have found the 12dB SINAD receive sensitivity of the sets to be close to the printed spec on 70cm band but the remaining background noise above that signal level drops off far slower than with other UHF sets I've tested.  They only get quiet once the LCD meter shows -95 (dBm)  (more like -100 on the test set)

**Comment by pietss** [comment]
Maybe I'll try tomorrow or in the next few days, thank you.
p.s. I see that there are many people who have two of these radios, I think it would be appreciated if they made known their assessments of the differences with the different firmwares they have tested, for example I use ijv on one and egzumer on the other, and when it happens to me, for the little I can do, I make known what I observe. Bye

**Comment by pietss** [comment]
I tested with kamilsss655, it seems to me that it is in third place, it is the least sensitive, maybe it could be a matter of setting something different but checking the menu nothing comes to mind. For the moment I keep ijv and egzumer on the two radios. I renew the invitation to those who own the two radios to do the tests, it does not take much

**Comment by tjcirillo** [comment]
i'm so glad i found this thread. i too compared egzumer v22 w IJV 2.9r4.  i have two radios, one running each so i can do side by side comparisons.  this started with an ISS pass. i easily hear it ascend, max elev, and descend however on the egzumer i missed 60% of the signal during the pass. i don't know if the Smeters (in dBA) are calibrated so i don't know if this is fair, but on the same signal, the IJV shows 10-15dB higher strength.  this on 2meter repeaters, 440, and NOAA wx.  and on scan, the IJV breaks squelch more often. on airband, when i (rarely) get a long enough signal to compare, the IJV is much more solid, and shows a higher signal strength.  same antennas, incidentally. last, i put IJV on both radios to make sure it wasn't a radio difference, and it's not.  with IJV on both they perform equal.  i don't have a signal generator so this is as far as i can go.  ps, i was rooting for egzumer as it is less complicated.

**Comment by pietss** [comment]
AM at around 133 MHz, continuous transmission of weather information, it seemed to me they were similar. As far as ijv is concerned, the dedication and regularity with which he makes the firmware available is to be admired but from the little I understand he seems rather reluctant to share the information; citation egzumer “IJV refuses to share his code, so I'm unable to even take a look for his improvements”. 


Now I will tell a short story. I'm off topic but I don't think the world will fall if I talk about something else for a moment. I will give my brief unsolicited opinion on the forums, often the problems encountered are similar even if the topics covered are different, and arise from the moderators. 
An Italian Facebook group - uv-k5 - was freely visible until a few days ago, but has now been privatised. One of the moderators had as a daily obligation the mockery of those who did not submit to the constraints of the regulation and therefore almost every day he said how sorry he was but he could not admit them into the group, a typical way of acting of hypocritical sadistic subjects who enjoy exercising the power. Naturally, this unbalanced person, not happy with this, applied himself daily and often several times in the same day with intimidating messages towards those who did not have an amateur radio certificate, repeating in an obsessive compulsive way that the radio in question could not be used. In essence he wants to exercise power over as many people as possible but at the same time, like a good tormentor, he floods with contemptuous and hostile messages if they are not radio amateurs. What is clearly ridiculous and what unmasks it is obviously the fact that it would have been enough for him to limit the group to amateur radio operators only, but this would have taken away the perverse pleasure of his intimidating and sadistic messages and then moved on to the latest rascality, namely the privatization of the group that has prevented many people like me who are not registered on facebook from being able to read. This last game was also made to try to encourage registrations and feed the infinite thirst for power (even if ridiculously limited in a facebook group) and continue with the harassment. Naturally these mentally disturbed people will be able to act freely as long as we bow to their will, this if it happens in almost insignificant areas like this it cannot do much damage and is negligible, it is sufficient to draw information from various other sources, but in my opinion it would be good for any so as to expose the techniques of these mean people

**Comment by tjcirillo** [comment]
i get your point.  

on the sensitivity, i've run further tests and they are inconclusive.  sometimes the egzumer performs the same as ijv.  just tested on ISS and the same

**Comment by ErikS-web** [comment]
Let's have a good laugh at the comment "_shows a higher signal strength_". 

Listen to the quality (Q factor). 
The S-meter is not the same on any receiver and with firmware you can fool it even more ;-)

**Comment by tjcirillo** [comment]
Very interesting! i hadn't considered a lot of what you discuss here.  i'd
been using a simple doppler adjustment of +/- .005 with success.  i test
the egzumer against the ijv and get varying results. maybe what you mention
gives some insight into why my results vary.  thanks, tom

**Comment by tjcirillo** [comment]
continuing this:
i took a nanovna and swept 2M freqs and 440 as well.  tight sweep.  and walked off two quansheng's, one with ijv and the other egz.  at both frequencies, they died at approximately the same distance from the nanovna.  they seemed to be equal.  a nanovna generates  a signal in the order of microwatts
